p4include/psa.p4

```diff
@@ -517,12 +526,15 @@ extern Checksum<W> {
   /// time the object is instantiated, that is, whenever the parser or control
   /// containing the Checksum object are applied.
   /// All state maintained by the Checksum object is independent per packet.
+  @packetState
```

Comment:
It looks to me like we may want this annotation to be on the extern itself rather than on each method.
Does it ever make sense to have only some methods of an extern be `@packetState`?